setzer/dialogs/document_wizard/pages/page_general_settings.py

```diff
@@ -128,7 +128,7 @@ def __init__(self):
         self.packages_rightbox = Gtk.VBox()
         self.packages_rightbox.set_size_request(348, -1)
         self.option_packages = dict()
-        self.option_packages['ams'] = Gtk.CheckButton.new_with_label('AMS math packages')
+        self.option_packages['ams'] = Gtk.CheckButton.new_with_label('ams')
```

Comment:
I guess this is okay here for consistency?

Reply:
No, this is not a single package, so it _should_ be inconsistent.

Reply:
Okay, will change.

Reply:
Please mark it as translatable too.